src/mscorlib/src/System/Runtime/Intrinsics/X86/Avx2.PlatformNotSupported.cs

```diff
@@ -722,7 +722,40 @@ public static class Avx2
         /// __m256i _mm256_inserti128_si256 (__m256i a, __m128i b, const int imm8)
         /// </summary>
         public static unsafe Vector256<ulong> Insert(Vector256<ulong> value, ulong* address, byte index) { throw new PlatformNotSupportedException(); }
-        
+
+        /// <summary>
+        /// __m256i _mm256_inserti128_si256 (__m256i a, __m128i b, const int imm8)
```

Comment:
These comments should be `__m256i _mm256_stream_load_si256 (__m256i const* mem_addr)`.

Reply:
Ah, my bad. Let me fix these comments later. Thank you!